config.json

```diff
@@ -1,8 +1,9 @@
 {
     "platform": "virtualbox",
-    "box_name": "playa_mesos_ubuntu_14.04",
+    "box_name": "mesos_026_ubuntu_14.04",
     "base_url": "http://downloads.mesosphere.io/playa-mesos",
     "ip_address": "10.141.141.10",
+    "mesos_release": "0.26.0-0.2.145.ubuntu1404",
```

Comment:
works for me. The versions inside the image where old and not compatible with docker 1.6+.
Docker up to 1.6 may fail to work with docker hub due to credentials revoke.